docs/controls/SlidableListItem.md

```diff
@@ -86,10 +81,9 @@ If you use **SlidableListItem** in a **ListView** with the **ItemClick** event,
 		<TextBlock Text="My Great Text" TextWrapping="NoWrap"/>            
 	</StackPanel>
 </controls:SlidableListItem> 
-
 ```
 
-## <a name="swipe"></a> Moving to SwipeControl
+## Moving to SwipeControl
```

Comment:
There are external links that would be broken by changing this, please revert back